examples.yaml

```diff
@@ -314,12 +314,9 @@ examples:
 
   - name: "Duration and timestamp"
     expr: |
-      // Needs Expr translation from CEL. Contributions welcome!
-      // https://github.com/polds/expr-playground/issues/43
-      //
       // Validate that 'expired' date is after a 'created' date plus a 'ttl' duration
-      has(object.expired) && 
-      timestamp(object.created) + duration(object.ttl) < timestamp(object.expired)
+      "expired" in object.expired && 
```

Comment:
Based on the input provided, `object.expired` is a string type. And in this case `in` operator will fail since it is not defined on string.

You can remove `expired` from `object` and it will check if `expired` key exists.

`"expired" in object`

Reply:
oh whoops. I had that local, just didn't push it. Good catch.